sycl/test/check_device_code/matrix/matrix-nvptx-bf16-test.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: cuda
+// REQUIRES: TEMPORARY_DISABLED
```

Comment:
```suggestion
// REQUIRES: cuda
// REQUIRES: TEMPORARY_DISABLED
```
This way we disable the test and keep old requirement in place, so it easier to recover original requirement.

@JackAKirk, if you expect to fix the issues with new pass manager in next few weeks, it look okay. Otherwise, it might be useful to disable these tests only when new PM is enabled (e.g. by adding a new feature to lit config file).

Reply:
I've made a fix and switched all tests back on: I simply allow `var1` from `var1 = reinterpret_cast<t>(var2);` to be named %call.ascast.i.i{{.*}}.i which covers all cases when using both the legacy and new pass managers.

Reply:
Great. @JackAKirk, please, update the title and the description of this PR.

Reply:
@mlychkov, FYI.

Reply:
> Great. @JackAKirk, please, update the title and the description of this PR.

Good point!